package.json

```diff
@@ -6,7 +6,7 @@
     "syntax"
   ],
   "homepage": "https://highlightjs.org/",
-  "version": "10.5.0",
+  "version": "10.5.1",
```

Comment:
This still needs to be reverted to 10.5.0.

Reply:
you don't bump patch for patches?  

ok sec

Reply:
my urge to call un-bumping "denting," though

Reply:
When we do another release we'll bump the version, not until then.

Reply:
It's likely to be 10.6 though, not 10.5.1... which is why I don't bump until I do an actual release and find out what the version number will be. :)